examples/cpp/HelloWorldExample/HelloWorldPublisher.cpp

```diff
@@ -80,7 +80,8 @@ bool HelloWorldPublisher::init(
             "PDP_PACKETS_TOPIC;" \
             "EDP_PACKETS_TOPIC;" \
             "DISCOVERY_TOPIC;" \
-            "PHYSICAL_DATA_TOPIC");
+            "PHYSICAL_DATA_TOPIC;" \
+            "MONITOR_SERVICE_TOPIC");
```

Comment:
apply this change here https://github.com/eProsima/Fast-DDS-statistics-backend/pull/207